NativeImport/ExampleNativeImportFeed.xml

```diff
@@ -79,6 +63,143 @@
                 <Featured>false</Featured>
                 <DisplayLocale>en_US</DisplayLocale>
             </Review>
+
+            <!-- sample review featuring attributes -->
```

Comment:
Similar to the basic review above, add a blurb about what separates this example from the other. Why would a client want to look at either?